app.rb

```diff
@@ -46,6 +46,34 @@
 Cuba.define do
   persist_session!
 
+  # We set the 'guests' layout as master template (file: /views/layout.guests.mote)
+  settings[:mote][:layout] = "layout.guests"
+
+  # Petition page
+  on "petition/:slug" do |slug|
```

Comment:
If I'm logged in as a campaigner and I got to `petition/:slug` will I still see a petition?

It seems like yes (the difference between `petition` and `petitions`, which I think is a good split) but I want to make sure that our campaigners can still see campaigns.